lib/controller/AuthController.js

```diff
@@ -96,10 +96,10 @@ AuthController.push({
     try {
       const user = await client.callback(request);
       await login(request, user);
-      return h.redirect(config.PUBLIC_PATH);
+      return h.redirect(`${config.PUBLIC_PATH}?login=1`);
```

Comment:
please document this as well.  the purpose of this flag isn't immediately clear, especially since it works in combination with frontend logic in `LoginState`.